### PR TITLE
Change the x86 tests to be able to run from the parent of tflm-micro directory

### PR DESCRIPTION
**Pull request by tanmaydas82**
BUG=b/246833632

### CODE REVIEW

File: tensorflow/lite/micro/tools/ci_build/test_x86.sh [hunk 1]
```diff
@@ -19,50 +19,52 @@
...
-ROOT_DIR=${SCRIPT_DIR}/../../../../..
+if [ ! -z ${1} ]; then
+  ROOT_DIR="${SCRIPT_DIR}/../../../../../.."
+else
+  ROOT_DIR="${SCRIPT_DIR}/../../../../.."
+fi
 cd "${ROOT_DIR}"
 
-source tensorflow/lite/micro/tools/ci_build/helper_functions.sh
+source ${1}tensorflow/lite/micro/tools/ci_build/helper_functions.sh
```
Comment (advaitjain) [inline tensorflow/lite/micro/tools/ci_build/test_x86.sh:29]: let's document the parameters ${1} and $[2} at top of script and assign them to named variables for the rest of the script.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -119,5 +119,5 @@ jobs:
       - name: Test
         run: |
           tensorflow/lite/micro/tools/ci_build/test_makefile.sh
-          tensorflow/lite/micro/tools/ci_build/test_x86.sh
+          tensorflow/lite/micro/tools/ci_build/test_x86.sh tflite-micro/
```
Comment (advaitjain) [inline .github/workflows/ci.yml:122]: let's run this script from the parent directory of tflite-micro

**Review by advaitjain — CHANGES_REQUESTED** [review]
looks goodl one nit only.

File: tensorflow/lite/micro/tools/ci_build/test_x86.sh (hunk 1 above)
Comment (tanmaydas82) [reply tensorflow/lite/micro/tools/ci_build/test_x86.sh]: Done

File: .github/workflows/ci.yml (hunk 2 above)
Comment (tanmaydas82) [reply .github/workflows/ci.yml]: Done

**Review by advaitjain — APPROVED** [review]

**Review by advaitjain — APPROVED** [review]

**Review by advaitjain — APPROVED** [review]

**Pull request merged**